Composer/packages/client/src/pages/design/PropertyEditorHeader.tsx

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React, { useMemo } from 'react';
+import { css } from '@emotion/core';
+import { FontSizes, NeutralColors } from '@uifabric/fluent-theme';
+import formatMessage from 'format-message';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+import { FontWeights } from '@uifabric/styling';
+
+const styles = {
+  errorLoading: css`
+    padding: 18px;
+  `,
+
+  propertyEditorHeaderStyle: css`
+    border-top: 1px solid ${NeutralColors.gray10};
+    padding: 0 18px;
+    margin-bottom: 0px;
+    border-bottom: 1px solid ${NeutralColors.gray60};
+  `,
+
+  title: css`
+    font-weight: ${FontWeights.semibold};
+    margin: 5px 0px;
+  `,
+
+  subtitle: css`
+    display: block;
+    height: 15px;
+    line-height: 15px;
+    font-size: ${FontSizes.size12};
+    color: ${NeutralColors.gray130};
+    font-weight: ${FontWeights.semibold};
+  `,
+
+  description: css`
+    margin-top: 0;
+    margin-bottom: 10px;
+    white-space: pre-line;
+    font-size: ${FontSizes.size12};
+  `,
+
+  helplink: css`
+    margin-top: 15px;
+    font-size: ${FontSizes.size12};
+  `,
+};
+
+export type PropertyEditorHeaderProps = {
+  projectData: { isRootBot: boolean; isRemote: boolean };
+  botName: string;
+  description?: string;
```

Comment:
is this gonna be used anywhere?

Reply:
There is actually no use of the property. I was hoping it could be something that @zhixzhan  passes to the component initially. But the static text is just inside the propertyeditorHeader component now